src/components/layout/GradientBottomActions.tsx

```diff
@@ -99,7 +104,7 @@ export const GradientBottomActions = ({
       ]}
     />
     <View style={styles.buttonContainer} pointerEvents="box-none">
-      {primaryAction}
+      {primaryAction && <ButtonSolid {...primaryAction}></ButtonSolid>}
```

Comment:
In this specific component, `ButtonSolid` should always be rendered with the `fullWidth` variant, then we can include it directly:
```suggestion
      {primaryAction && <ButtonSolid fullWidth {...primaryAction}></ButtonSolid>}
```